Cilsil.Test/TestRunManager.cs

```diff
@@ -1,42 +1,40 @@
 ﻿// Copyright (c) Microsoft Corporation.
```

Comment:
weird formatting on this file, spaces, curly brace newlines, using orders..

Reply:
thanks seems fixed after rerunning editorconfig.